tests/e2e/kine/Vagrantfile

```diff
@@ -0,0 +1,157 @@
+ENV['VAGRANT_NO_PARALLEL'] = ENV['E2E_STANDUP_PARALLEL'] ? nil : 'no'
+NODE_ROLES = (ENV['E2E_NODE_ROLES'] ||
+  ["server-0"])
```

Comment:
Let's change the default to 3 server if we want to run with three servers

Reply:
You can't run it in GHA then

Reply:
The nightly should set the variables to different values, with the GHA running the default of one.